go/kbfs/libkbfs/kbfs_ops_test.go

```diff
@@ -186,9 +186,8 @@ func kbfsOpsInit(t *testing.T) (mockCtrl *gomock.Controller,
 	return mockCtrl, config, ctx, cancel
 }
 
-func kbfsTestShutdown(
-	t *testing.T, mockCtrl *gomock.Controller, config *ConfigMock,
-	ctx context.Context, cancel context.CancelFunc) {
+func kbfsTestShutdown(ctx context.Context,
```

Comment:
Small quibble to have `ctx` on the next line down, and keep the rest of the params on lines less than 80 chars.